netket/operator/_pauli_strings.py

```diff
@@ -493,6 +487,19 @@ def gccf_fun(x, sections):
         return jit(nopython=True)(gccf_fun)
 
 
+def _count_of_locations(of_qubit_operator):
+    # we always start counting from 0, so we only determine the maximum location
+    def max_or_default(x):
+        x = list(x)
+        return max(x) if len(x) > 0 else -1  # -1 is default
+
+    n_qubits = max_or_default(
+        max_or_default(term[0] for term in op) for op in of_qubit_operator.terms.keys()
+    )
+    n_qubits += 1
```

Comment:
```suggestion
    n_qubits = 1 + max_or_default(
        max_or_default(term[0] for term in op) for op in of_qubit_operator.terms.keys()
    )
```

unimportant, but i prefer this style